spec/support/api/schemas/service_day.json

```diff
@@ -30,6 +30,7 @@
     },
     "total_time_in_care": {
       "type": [
+        "integer",
```

Comment:
I don't think this should be necessary; can we walk through the spec that's throwing this error together?  I've seen it before and it's usually a symptom of another problem.